### PR TITLE
Rails project creation

### PR DESCRIPTION
**Pull request by HarryPerez**
## Screenshots
Captura de Pantalla 2019-05-21 a la(s) 17 12 18

## Trello card
https://trello.com/c/V5N3CkyQ/14-crear-aplicaci%C3%B3n-rails-wbooksapi

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+ruby-2.3.7
```
Comment (Gmarn) [inline .ruby-version]: End of line ¬¬

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.3.7'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.2.3'
```
Comment (Gmarn) [inline Gemfile]: Hum, In the card is specify rails 5.1 ¬¬

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,74 @@
+class PostsController < ApplicationController
```
Comment (Gmarn) [inline app/controllers/posts_controller.rb]: I think you go to far in the tutorial you used ;)
Can you rollback every files scaffolds created for you?
you will need to, rollback migration and remove: migration file, models for post, controllers, test , routes, views and CSS ;)

File: Gemfile (hunk 2 above)
Comment (leamotta) [reply Gemfile]: @HarryPerez so' loco vo'?

Comment (HarryPerez) [reply Gemfile]: @leamotta que hace aca vo.. deberias estar en ⚽️ en el ✈️

**Pull request merged**